channels/pe.m3u

```diff
@@ -13,22 +13,24 @@ https://tv.siete.us/bestcablesports/bestcablesports/index.m3u8
 https://video1.getstreamhosting.com:1936/8106/8106/playlist.m3u8
 #EXTINF:-1 tvg-id="BHTV.pe" tvg-name="BHTV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://i.imgur.com/7hP4nps.png" group-title="General",BHTV (720p)
 http://cdn1.ujjina.com:1935/iptvbhtv/livebhtvtv/playlist.m3u8
-#EXTINF:-1 tvg-id="CajamarcaTV.pe" tvg-name="Cajamarca TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CajamarcaTV/picture?width=320&height=320" group-title="Local",Cajamarca TV (480p)
-https://live.siete.us/cajamarcatv/cajamarcatv/index.m3u8
+#EXTINF:-1 tvg-id="CajamarcaTV.pe" tvg-name="Cajamarca TV" tvg-country="PE" tvg-language="Spanish" tvg-logo="https://graph.facebook.com/CajamarcaTV/picture?width=320&height=320" group-title="Local",Cajamarca TV (480p) [Not 24/7]
+https://tv.siete.us/cajamarcatv/cajamarcatv/index.m3u8
```

Comment:
Isn't responding

Reply:
The channel was accessible at https://cajamarcatv.pe/tv, says "24-hour channel" but I add Not 24/7, will be accessible in the future.